source/main.cpp

```diff
@@ -55,11 +56,34 @@ std::string FormatDeviceCaps(vk::PhysicalDevice PhysicalDevice);
 
 int main(int argc, char* argv[])
 {
+	std::ifstream ifs("settings.json");
+    Json::Reader reader;
+    Json::Value obj;
+    reader.parse(ifs, obj);
+
+    int render_size_x = obj["render_size_x"].asInt();
+	if (render_size_x == 0){
```

Comment:
Be sure to run `.clang-format` on your code to fix style issues like this. I use [Allman-style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) bracing. `clang-format` will fix all of that for you.

Reply:
Fixed in [bbc8649](https://github.com/Wunkolo/vkblam/pull/1/commits/bbc8649cf4c93550cbd1a16611d7e3a3c108c38b). Closing.